services/site/src/app/layout.tsx

```diff
@@ -0,0 +1,60 @@
+import "app/styles/nprogress.scss";
+import "app/styles/global.scss";
+import "app/styles/fonts.scss";
+import "app/styles/custom.scss";
+import "focus-visible/dist/focus-visible";
+
+import { loadDevMessages, loadErrorMessages } from "@apollo/client/dev";
+import { getClientConfig } from "app/lib/config/rsc";
+import { Metadata } from "next";
+import Script from "next/script";
+
+import ClientProviders from "./ClientProviders";
+
+const RootLayout: React.FunctionComponent<React.PropsWithChildren> = ({ children }) => {
+  const plausibleUrl = process.env.SITE_PLAUSIBLE_BASE_URL;
+  const domain = process.env.SITE_HOST;
+
+  loadDevMessages();
+  loadErrorMessages();
+
+  return (
+    <html lang="en">
+      <head>{plausibleUrl && <Script data-domain={domain} src="/js/script.js" />}</head>
+      <body>
+        <ClientProviders config={getClientConfig()}>{children}</ClientProviders>
+      </body>
+    </html>
+  );
+};
+
+export default RootLayout;
+
+export const metadata: Metadata = {
```

Comment:
This metadata will be the default value for each of the pages in the app directory. So we only have to specify the values that should be different in the individual pages. That will allow us to get rid of the duplication that we currently have in the pages.

Reply:
Can you please use this new way of displaying metadata on all pages we have where meta info for SEO is used. If that was done already, nice! its just so many changes maybe I overlooked it.

Reply:
Unfortunately we can't use this for the pages we already have, since this feature is only available with the "app router" and those use the "pages router". While migrating the existing pages to app router pages we can also migrate the metadata and remove the duplication there.